### ISSUE TITLE
Only prevent Quelling/Bfury from cutting mk trees

### ISSUE THREAD

**Issue by Draze22** [arcadia-redux/12v12#609]
Quelling Blade and Battle Fury the only things we can be sure are being used to grief Monkey King when they cut down his tree. Preventing other stuff like Ball Lightning from being cast leads to situations like this:

![image](https://user-images.githubusercontent.com/63279022/179003363-92753146-fcea-4054-80f4-dbef8ac00fc6.png)

**Comment by AustraliaIsMyCity** [comment]
eh, do we really have to? coz like if the errors happening, mk's getting cut off his tree, and i used to see a lot more reports of that than i see people complaining about being unable to cast because of mk tree disable. might just disable it for storm ulti.

**Comment by Draze22** [comment]
Yes we have to, Monkey King players picked the hero knowing this could happen, their team didn't sign up for getting their spell casting griefed by their own team mate.

**Comment by Draze22** [comment]
Also all players who are griefing can now be kicked which covers players who are intentionally using abilities to grief Monkey King.

**Comment by AustraliaIsMyCity** [comment]
wait why dont we just make this only work if help is disabled

**Comment by Draze22** [comment]
> wait why dont we just make this only work if help is disabled

Doesn't stop the rare occasion where the MK happens to have help disabled from Storm Spirit. I don't see any way this system will ever prevent griefing without also screwing over people who are just playing the game.
